printing-services/backend/.env

```
@@ -0,0 +1,23 @@
+# Development Environment Variables
+DATABASE_URL="file:./dev.db"
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)